homeassistant/components/homeconnect/config_flow.py

```diff
@@ -0,0 +1,22 @@
+"""Config flow for Home Connect."""
+import logging
+
+from homeassistant import config_entries
+from homeassistant.helpers import config_entry_oauth2_flow
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+@config_entries.HANDLERS.register(DOMAIN)
+class OAuth2FlowHandler(config_entry_oauth2_flow.AbstractOAuth2FlowHandler):
```

Comment:
Pass the domain as keyword argument domain and remove the decorator.

https://developers.home-assistant.io/docs/config_entries_config_flow_handler#defining-your-config-flow